src/elements/partners-block.html

```diff
@@ -5,8 +5,6 @@
 <link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
 
 <link rel="import" href="../redux-mixin.html">
-<link rel="import" href="./shared-styles.html">
```

Comment:
shared styles file is important from parent component, so it's already have it. Do you think it's better to include this in each component?